perceval/backends/core/jira.py

```diff
@@ -273,8 +273,8 @@ class JiraClient(HttpClient):
 
     :param URL: URL of the JIRA server
     :param project: filter issues by project
-    :param user: JIRA's username
-    :param password: JIRA's password
```

Comment:
same as the above comment, revert this change for now and change it only after implementation